modules/apigee_edge_teams/src/Entity/TeamInvitation.php

```diff
@@ -190,7 +190,9 @@ public function setStatus(int $status): TeamInvitationInterface {
    * {@inheritdoc}
    */
   public function getRecipient(): ?string {
-    return $this->get('recipient')->value;
+    $recipientEmail = $this->get('recipient')->value;;
```

Comment:
There is an extra `;` at the end of the line.

Reply:
Thanks